var/spack/repos/builtin/packages/gdk-pixbuf/package.py

```diff
@@ -43,7 +45,7 @@ class GdkPixbuf(Package):
     depends_on('jpeg')
     depends_on('libpng')
     depends_on('zlib')
-    depends_on('libtiff')
+    depends_on('libtiff@:3.9.7', when='+tiff')
```

Comment:
While I agree that passing tests are a good thing, do you think it's worth it to keep using an outdated (and potentially vulnerable) version of libtiff for it? gdk-pixbuf also seems to work with newer versions, right?